package.json

```diff
@@ -26,6 +26,7 @@
     "es6-promise": "^2",
     "eslint": "^1.9.0",
     "eslint-plugin-no-unsafe-innerhtml": "^1.0.14",
+    "eslint-plugin-no-wildcard-postmessage": "0.1.2",
```

Comment:
I'm a bit concerned that you didn't ask for review on this code initially..

I see that it lacks the Gaia code style (especially 2-space indentation). Morever it lacks a license.